conf/taiga/local.py

```diff
@@ -9,7 +9,8 @@
 # https://github.com/taigaio/taiga-back/blob/master/settings/common.py
 
 from .docker import *
+import os
 
 PUBLIC_REGISTER_ENABLED = False
-DEBUG = True
-TEMPLATE_DEBUG = False
+DEBUG = os.getenv('DEBUG').lower() == 'true'
```

Comment:
:+1: The casting... :smile:

Reply:
Ah yes. I ran into that on the LDAP effort and forgot to port that fix back to this PR. :+1: